### PR TITLE
version up Version 0.4.5 and add .spec file

### PR DESCRIPTION
**Pull request by SergeyMy**
version up Version 0.4.5 and add .spec file

### CODE REVIEW

File: simplescreenrecorder.spec [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+%undefine __cmake_in_source_build
+
+Name:           simplescreenrecorder
+Version:        0.4.5
+Release:        1%{?dist}
+Summary:        Simple Screen Recorder is a screen recorder for Linux
+
+License:        GPLv3
+URL:            https://www.maartenbaert.be/simplescreenrecorder/
+Source0:        https://github.com/SergeyMy/ssr/archive/%{version}/%{name}-%{version}.tar.gz
```
Comment (mathieu-aubin) [inline simplescreenrecorder.spec:12]: Is this supposed to point to your fork or the main project's repository?

Comment (SergeyMy) [reply simplescreenrecorder.spec:12]: > Is this supposed to point to your fork or the main project's repository?

main project's repository
only a label other than master is needed!
:)

Comment (Creteil) [reply simplescreenrecorder.spec:12]: What command we need to use to fetch this « tag » revision of code ?

Comment (SergeyMy) [reply simplescreenrecorder.spec:12]: :)
git clone https://github.com/SergeyMy/ssr.git -b 0.4.5
or
wget https://github.com/SergeyMy/ssr/archive/0.4.5/simplescreenrecorder-0.4.5.tar.gz

**Comment by petterreinholdtsen** [comment]
I too hope @MaartenBaert will wrap up a new release soon, as it would make life easier when maintaining the Debian package.  But I suspect this pull request would be easier to handle if it was split in two, one for the new release and a separate one for adding a rpm spec file.

**Pull request closed without merging**